corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -110,11 +117,41 @@ def _process_main_forms(self):
             form_received = wrapped_form.received_on
             assert last_received_on <= form_received
             last_received_on = form_received
-            try:
-                self._migrate_form_and_associated_models(wrapped_form)
-            except Exception:
-                self.log_error("Unable to migrate form: {}".format(change.id))
-                raise
+
+            case_doc = wrapped_form.form.get('case')
```

Comment:
forms can update multiple cases. You can use `get_case_ids_from_form` to get all cases touched by a form.